### PR TITLE
#120: API-core-policies-MDRangePolicy from md to rst

### PR DESCRIPTION
**Pull request by antoinemeyer5**
### THIS PR:
* API/core/policies/MDRangePolicy transition from .md to .rst

- [x] uses cppkokkos
- [x] rebased on latest main 

before (.md) | after (.rst)
:----------------: | :-------------:
before_1 | image
before_2 | image
before_3 | image

### CODE REVIEW

**Review by fnrizzi — APPROVED** [review]

File: docs/source/API/core/policies/MDRangePolicy.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+Public Class Members
+--------------------
+
+Constructors
+~~~~~~~~~~~~
+
+.. cpp:function:: MDRangePolicy()
+    
+* Default Constructor uninitialized policy.
```
Comment (fnrizzi) [inline docs/source/API/core/policies/MDRangePolicy.rst:73]: please indent

File: docs/source/API/core/policies/MDRangePolicy.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+.. cpp:function:: MDRangePolicy(const Kokkos::Array<int64_t,rank>& begin, const Kokkos::Array<int64_t,rank>& end)
+
+* Provide a start and end index.
```
Comment (fnrizzi) [inline docs/source/API/core/policies/MDRangePolicy.rst:77]: please indent

File: docs/source/API/core/policies/MDRangePolicy.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+   
+.. cpp:function:: MDRangePolicy(const Kokkos::Array<int64_t,rank>& begin, const Kokkos::Array<int64_t,rank>& end,  const Kokkos::Array<int64_t,rank>& tiling)
+
+* Provide a start and end index as well as the tiling dimensions.
```
Comment (fnrizzi) [inline docs/source/API/core/policies/MDRangePolicy.rst:81]: please indent

File: docs/source/API/core/policies/MDRangePolicy.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+ 
+.. cpp:function:: template<class OT, class IT, class TT> MDRangePolicy(const std::initializer_list<OT>& begin, const std::initializer_list<IT>& end)
+
+* Provide a start and end index. The length of the lists must match the rank of the policy.
```
Comment (fnrizzi) [inline docs/source/API/core/policies/MDRangePolicy.rst:85]: please indent

File: docs/source/API/core/policies/MDRangePolicy.rst [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+* Provide a start and end index as well as the tiling dimensions.
+ 
+.. cpp:function:: template<class OT, class IT, class TT> MDRangePolicy(const std::initializer_list<OT>& begin, const std::initializer_list<IT>& end)
+
+* Provide a start and end index. The length of the lists must match the rank of the policy. 
+
+.. cpp:function:: template<class OT, class IT, class TT> MDRangePolicy(const std::initializer_list<OT>& begin, const std::initializer_list<IT>& end,  std::initializer_list<TT>& tiling)
+
+* Provide a start and end index as well as the tiling dimensions. The length of the lists must match the rank of the policy.
```
Comment (fnrizzi) [inline docs/source/API/core/policies/MDRangePolicy.rst:89]: please indent

**Comment by antoinemeyer5** [comment]
> see my comments

indentations made and screenshots updated !

File: docs/source/API/core/policies/MDRangePolicy.rst [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+* Determines the rank of the index space as well as in which order to iterate over the tiles and how to iterate within the tiles. ``outer`` and ``inner`` can be ``Kokkos::Iterate::Default``, ``Kokkos::Iterate::Left``, or ``Kokkos::Iterate::Right``.
+
+
+Public Class Members
+--------------------
+
+Constructors
+~~~~~~~~~~~~
+
+.. cpp:function:: MDRangePolicy()
```
Comment (fnrizzi) [inline docs/source/API/core/policies/MDRangePolicy.rst:69]: please use cppkokkos here and below

**Review by fnrizzi — APPROVED** [review]

**Pull request merged**